src/udl/skel/PHP/pylib/lang_php.py

```diff
@@ -133,14 +133,39 @@ def _functionCalltipTrigger(self, ac, pos, DEBUG=False):
             print "Arg separater found, looking for start of function"
         # Move back to the open paren of the function
         paren_count = 0
-        p = pos
-        min_p = max(0, p - 200) # look back max 200 chars
-        while p > min_p:
-            p, c, style = ac.getPrecedingPosCharStyle(ignore_styles=self.comment_styles)
-            if style == self.operator_style:
+        # Account for parenthesis at current position
+        p, c, style = ac.getCurrentPosCharStyle()
```

Comment:
If it's not a huge pain, can you change `p` to `pos` and `c` to `char`, assuming that's correct.  I know you didn't write that but it would be good to make it more clear while we're here.